requirements/deploy.txt

```diff
@@ -160,7 +160,7 @@ django-ipware==5.0.2
     # via
     #   -r requirements/pip.txt
     #   django-structlog
-django-polymorphic==3.1.0
+django-polymorphic==4.1.0
```

Comment:
It seems there is nothing breaking as far as I can tell https://github.com/jazzband/django-polymorphic/releases/tag/v4.1.0